.buildkite/pipeline.verify.yml

```diff
@@ -5,6 +5,7 @@ env:
   # TODO: Figure out a way for this to be sourced from our
   # rust-toolchain file
   RUST_VERSION: 1.62.0
+  PYTHON_VERSION: 3.10.5
```

Comment:
if it's gonna be duplicated from .python-version, would be nice to have a 
`# also update .python-version and the same in .python-version`